### DISCUSSION TITLE
Manager Grid EPS

### DISCUSSION THREAD

**Discussion by argwfm** [Security-Onion-Solutions/securityonion#9752] [Unsupported Versions] [answered]
Is anyone else's Grid showing a manager node w/ EPS of 0/0? Distributed v2.3.200 deployment here w/ a manager + receiver node. Manager status is OK and consuming events per Grafana but Grid isn't reflecting.

![image](https://user-images.githubusercontent.com/99749555/217846493-c2fcf160-2de4-4bdc-9ea4-b190560862da.png)

**Comment by InfosecGoon** [comment]
Does everything look normal in Grafana for those nodes? That is, does it show the expected level of traffic on the management ports and in the ingestion pipeline?

**Reply by argwfm** [reply]
Yes, everything looks good via Grafana for the manager. Checked the raw output of `/sensoroniagents/api/grid` and the consumption EPS is 0 as reflected w/in the Grid UI. Manager redis queue is showing ~100 event mean over the last 7 days.

**Reply by argwfm** [reply]
Just wanted to report this issue still persists w/ a fresh install using the latest SOv2.4.30 ISO.
Since upgrading from SOv2.3 forward nodes are no longer showing EPS via the Grid.
Please LMK if I can provide any additional data to support.

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/99749555/4d0b4bac-9ff5-4d82-ac09-32a673caf7be)

**Reply by dougburks** [reply, marked as answer]
In 2.3, there were both Production EPS and Consumption EPS:
https://docs.securityonion.net/en/2.3/grid.html

In 2.4, there is only Consumption EPS:
https://docs.securityonion.net/en/2.4/grid.html

Your forward nodes are producing logs and not consuming them and so their EPS is 0.

**Reply by argwfm** [reply]
Thanks @dougburks; missed that change. Assume there's no plans of reintroducing the production EPS for the forward nodes? Was using that as the go to for monitoring the health of the forward nodes and dependent TAP infrastructure.

**Reply by dougburks** [reply]
> Assume there's no plans of reintroducing the production EPS for the forward nodes? 

There are no plans of reintroducing that as it relied on functionality in 2.3 that is no longer present in 2.4.

> Was using that as the go to for monitoring the health of the forward nodes and dependent TAP infrastructure.

If you go to SOC Dashboards, the default Overview dashboard has an observer.name table that should show how many logs are coming from each forward node:
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/1ac8c4d7-4445-40fe-8661-94005c08d65d)
